src/components/shared/forms/fileUpload.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+
+import './styles/fileUpload.scss';
+
+export class FileUpload extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.singleFileRef = React.createRef();
+  }
+
+  singleFileRefClicked = () => {
+    this.singleFileRef.current.click();
+  }
+
+  onchange = (e) => {
+    this.props.onChange(e);
+  }
+
+  render() {
+    return <div className="file-uploader-container">
```

Comment:
nit: wrap the return statement.
```suggestion
    return (
       <div className="file-uploader-container">
           ...
       </div>
     );
```